evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -682,4 +685,220 @@ mod tests {
         let output = String::decode(&output).unwrap();
         assert_eq!(output, msg);
     }
+
+    #[test]
+    fn test_artifact_parsing() {
+        let s = r#"{
```

Comment:
shall we put this in `evm/test-data/`? cautious of bloating source files

Reply:
yeh let's do that